web/client/components/map/leaflet/__tests__/Map-test.jsx

```diff
@@ -169,7 +169,7 @@ describe('LeafletMap', () => {
 
         const leafletMap = map.map;
         expect(leafletMap.getZoom()).toBe(12);
-        map.setProps({zoom: 12, center: {y: 44, x: 10}});
+        map.setProps({zoom: 12.4, center: {y: 44, x: 10}});
         expect(leafletMap.getZoom()).toBe(12);
```

Comment:
This test is to "check if the map changes when receive new props"
The zoom is 12 before the test, so you should expect another value after the setProps().